js.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+function daysInYear(year) {
+  if (Number.isInteger(year)) {
+      let feb = new Date(year, 1, 29).getDate();
+      if (feb === 29) {
+          return console.log(366);
+      }
+      else {
+          return console.log(365);
+      }
+  }
+  else
+      return (console.log('Exception'));
```

Comment:
Based on the requirement we should `throw an exception`, not return undefined (this code returns undefined)